scripts/docker-compose-config.bash

```diff
@@ -17,12 +17,25 @@ show_error() {
 
 
 env_file=".env"
+no_interpolate=false
+
```

Comment:
Please propagate any changes back to https://github.com/ITISFoundation/osparc-simcore/blob/master/scripts/docker/docker-compose-config.bash :--) with a PR thx

Reply:
I reverted the changes